crates/aptos-dkg/src/pvss/weighted/weighted_config.rs

```diff
@@ -130,7 +124,6 @@ impl WeightedConfig {
     ///
     /// Returns the index of this player's share in the vector of shares, or None if out of bounds.
     pub fn get_share_index(&self, i: usize, j: usize) -> Option<usize> {
-        assert_lt!(i, self.tc.n);
```

Comment:
Why remove this?

Was this wrong actually? `self.tc.n` is the total # of shares, not players...

Reply:
yup, if everyone gets 1 share except that validator 0 gets 0 shares, `self.tc.n == n-1` and `i` can be `n-1` too.

Reply:
then should we assert `assert_le`?

Reply:
we are allowing num_players > num_shares, so we shouldn't.